### DISCUSSION TITLE
Can't remap <C-i>?

### DISCUSSION THREAD

**Discussion by Lekyaira** [NvChad/NvChad#3122] [Q&A] [answered]
I'm attempting to remap `<C-i>` to swap to the right pane. 
`vim.keymap.set("n", "<C-i>", "<C-w>l")`

This works well for all the other directions (I'm using `<C-h>`, `<C-n>`, and `<C-e>`) but `<C-i>` doesn't get remapped and instead moves to the next tab. Doing a `vim.keymap.del("n", "<C-i>")` throws an error that the mapped key doesn't exist. Moving the unmap to after I map `<C-i>` doesn't throw an error, so it seems like the unmap is working as intended.

Is this a bug? Is it getting overridden elsewhere in NvChad?

**Comment by Lekyaira** [comment, marked as answer]
I've tracked the issue down to tmux. Works great when I'm not in a tmux session, so it's not an NvChad issue at all.

**Reply by siduck** [reply]
@Lekyaira  you can disable <Tab> i.e use vim.keymap.del and then map <C-i>
